Makefile

```diff
@@ -280,6 +284,7 @@ e2e-stop-instrumented:
 
 .PHONY: e2e-test-coverage
 e2e-test-coverage: E2E_TEST_ARGS = --json-report=report_e2e.json --output-dir=.
+e2e-test-coverage: E2E_TEST_CODE_ARGS = --compliance-api-port=8385
```

Comment:
what is diff 8385 vs 8384

Reply:
So if you have the Kind cluster running, it exposes the 8384 port so it can't be reused when running `e2e-test-coverage` since the controller is run outside of the cluster.